pages/enterprise/changelog.rst

```diff
@@ -2,6 +2,63 @@
 Changelog
 *********
 
+Graylog Enterprise 2.2.3
+========================
+
+Released: 2017-04-04
+
+Plugin: Archive
+---------------
+
+Metadata is now stored in MongoDB
```

Comment:
These should probably be bullet lists, so that each item is shown on its own line.

http://www.sphinx-doc.org/en/stable/rest.html#lists-and-quote-like-blocks